packages/curvenote-cli/src/submissions/submit.utils.ts

```diff
@@ -383,36 +381,55 @@ export async function checkVenueSubmitAccess(session: ISession, venue: string) {
   }
 }
 
-export async function getVenueCollections(session: ISession, venue: string) {
-  try {
-    const collections = (await getFromJournals(
-      session,
-      `sites/${venue}/collections`,
-    )) as CollectionListingDTO;
-
-    const openCollections = collections.items.filter((c) => c.open);
-
-    if (openCollections.length === 0) {
-      session.log.info(`${chalk.red(`🚦 venue "${venue}" is not accepting submissions.`)}`);
-      process.exit(1);
-    }
+/**
+ * Fetch `venue` collections from API
+ */
+export async function listCollections(
+  session: ISession,
+  venue: string,
+): Promise<CollectionListingDTO> {
+  return getFromJournals(session, `sites/${venue}/collections`);
+}
 
-    if (openCollections.length > 1) {
-      session.log.info(
-        `${chalk.green(`💚 venue "${venue}" is accepting submissions in the following collections: ${openCollections.map((c) => collectionMoniker(c)).join(', ')}`)}`,
-      );
-    } else {
-      session.log.info(
-        `${chalk.green(`💚 venue "${venue}" is accepting submissions (${collectionMoniker(openCollections[0])}).`)}`,
-      );
-    }
-    return collections;
+/**
+ * Get collections from `venue` and log information about open collections
+ *
+ * This will fail with `process.exit(1)` if the fetch for venue collections fails.
+ * By default, it also fails if there are no open collections.
+ *
+ * If `requireOpenCollections` is false, this function will not fail if there are
+ * only closed collections or no collections at all.
+ */
+export async function getVenueCollections(
+  session: ISession,
+  venue: string,
+  requireOpenCollections = true,
```

Comment:
This parameter is new. This allows us to get a venue's collections, even if they are all closed or it's simply an empty list. Previously, both of those would cause `process.exit(1)`; now, that is still the default case, but setting `requireOpenCollections: false` allows both those cases to pass and still return the collections.

Reply:
ok cool and it is set to false in `check` but left as default `true` in `submit` 👍🏼

So in terms of the action, this means for no open collections the check will pass but then a submit will fail if collections are closed. seems reasonable / better!